### PR TITLE
Add OurShoppingList to the Task Management & To-do Lists section

### PR DESCRIPTION
**Pull request by nanawel**
~~As for #3258, sorry for the mess with `AUTHORS.md` but I just used `make contrib` as required in the contribution rules :confused:~~

OK fixed, I've only changed `README.md`.

### CODE REVIEW

**Comment by KieranRobson** [comment]
make contrib isnt needed for the PRs, just the checklist in the PR template is required
Ill get these both reviewed sometime today!

**Comment by nanawel** [comment]
Hi, thanks for your time on this. I'll change the PR to make it shorter then.

**Comment by nanawel** [comment]
That should be okay now :slightly_smiling_face:

File: README.md [hunk 1]
```diff
@@ -1928,6 +1928,7 @@ _Related: [Software Development - Project Management](#software-development---pr
 - [Minimalist Online Markdown Editor](http://markdown.pioul.fr) - The simplest and slickest online Markdown editor. ([Source Code](https://github.com/pioul/Minimalist-Online-Markdown-Editor)) `MIT` `Nodejs`
 - [myTinyTodo](https://www.mytinytodo.net/) - Simple way to manage your todo list in AJAX style. Uses PHP, jQuery, SQLite/MySQL. GTD compliant. ([Demo](https://www.mytinytodo.net/demo/), [Source Code](https://github.com/maxpozdeev/mytinytodo/)) `GPL-2.0` `PHP`
 - [Nullboard](https://github.com/apankrat/nullboard) - Single-page minimalist kanban board; compact, highly readable and quick to use. `BSD-2-Clause` `Javascript`
+- [Our Shopping List](https://github.com/nanawel/our-shopping-list) - Simple shared list application. Typical uses include shopping lists of course, and any other small todo-list that needs to be used collaboratively. ([Demo](https://osl.lanterne-rouge.info/) ) `AGPL-3.0` `Nodejs`
```
[the remark this answers is not in the document]
Comment (nanawel) [reply README.md]: Thanks @KieranRobson, that should be fixed now.

**Review by nodiscc — APPROVED** [review]

**Pull request merged**